src/platforms/dotnet/guides/aspnet/troubleshooting.mdx

```diff
@@ -31,4 +31,17 @@ using System.Net;
 ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12
 ```
 
+### Tags not showing if set on Application_Start
+
+That happens because the `scope` from `requests` will be different from the one executed during the execution of Application_Start.
```

Comment:
```suggestion
This happens because the `scope` from `requests` will be different from the one executed during the execution of Application_Start.
```